tests/vmipreset_test.go

```diff
@@ -345,9 +363,8 @@ var _ = Describe("[rfe_id:609][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			err := virtClient.RestClient().Post().Resource("virtualmachineinstancepresets").Namespace(tests.NamespaceTestDefault).Body(overridePreset).Do().Error()
 			Expect(err).ToNot(HaveOccurred())
 
-			By("Waiting for preset to be created")
-			getPreset(virtClient, overridePrefix)
-			Expect(err).ToNot(HaveOccurred())
+			// Give virt-api's cache time to sync before proceeding
+			time.Sleep(3 * time.Second)
```

Comment:
hey, obviously based on all my previous feedback I'm in favor of the sleep here. but out of curiosity, when you removed the "getPreset", did the test occasionally fail before the sleep was added?

Reply:
yes *cough*

Reply:
Overall I really dislike blind sleeps, but you make a good point. Doing it this way makes it glaringly obvious that there's a hacky workaround afoot.